traffic_ops/testing/api/v14/divisions_test.go

```diff
@@ -16,19 +16,42 @@ package v14
 */
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
 func TestDivisions(t *testing.T) {
-	WithObjs(t, []TCObj{Parameters, Divisions}, func() {
+	WithObjs(t, []TCObj{Parameters, Divisions, Regions}, func() {
+		TryToDeleteDivision(t)
 		UpdateTestDivisions(t)
 		GetTestDivisions(t)
 	})
 }
 
+func TryToDeleteDivision(t *testing.T) {
+	division := testData.Divisions[0]
+	resp, _, err := TOSession.DeleteDivisionByName(division.Name)
+
+	if err == nil {
+		t.Errorf("should not be able to delete a division prematurely")
+		return
+	}
+
+	if strings.Contains(err.Error(), "Resource not found.") {
+		t.Errorf("division with name %v does not exist", division.Name)
+		return
+	}
+
+	if strings.Contains(err.Error(), "This division is currently used by regions.") {
```

Comment:
The test doen't compile, but it looks like this won't pass, needs to be `cannot delete division because it is being used by a region`.

Reply:
Looks like I need to use `DeleteRegionByID`. Should I use a status code instead of looking at the error message so that this is less brittle? A bad request does not necessarily indicate the error I'm actually testing for, so I'm not sure what the best way is.